src/providers/codeactions.ts

```diff
@@ -172,16 +132,64 @@ export class CodeActions {
                     break
                 }
                 fixString = regexResult[1]
-                replaceRangeWithString(document, range, fixString)
+                this.replaceRangeWithString(document, range, fixString)
                 break
             case 45:
-                replaceMathDelimitersInRange(document, range, '$$', '\\[', '\\]')
+                this.replaceMathDelimitersInRange(document, range, '$$', '\\[', '\\]')
                 break
             case 46:
-                replaceMathDelimitersInRange(document, range, '$', '\\(', '\\)')
+                this.replaceMathDelimitersInRange(document, range, '$', '\\(', '\\)')
```

Comment:
We should not call `texMathEnvFinder` for `$`. It does not work.

Reply:
For standard expressions like

```latex
Some maths $1 + 3 =
4$ within text
```

 it is working. Can you give a failing example?

Reply:
I mean, it is not intended behavior. The behavior might change in the future refactoring.

Reply:
Fine, we can drop the support for `$...$` split over several lines. This is consistent with the hovering feature.